docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -10,54 +10,51 @@ When you've finished your workpad, you can share it outside of {kib}.
 
 Create a JSON file of your workpad that you can export outside of {kib}.
 
-. From your workpad, click the *Share workpad* icon.
+Click *Share > Download as JSON*.
 
-. Select *Download as JSON*.
-+
-[role="screenshot"]
-image::images/canvas-export-workpad.png[Export single workpad]
+//Need to replace image
+//[role="screenshot"]
+//image::images/canvas-export-workpad.png[Export single workpad]
 
-Want to export multiple workpads? Go to the *Canvas workpads* view, select the workpads you want to export, then click *Export*.
+Want to export multiple workpads? Click *Canvas*, select the workpads you want to export, then click *Export*.
```

Comment:
Do you mean the `Canvas` link in the breadcrumbs up top or in the left sidebar? Do we need to make that distinction here?

Reply:
Fixed!